visidata/guides/DirSheet.md

```diff
@@ -0,0 +1,35 @@
+---
+sheettype: DirSheet
+---
+# Directory Sheet
+
+The **DirSheet** is a display of files and folders in the *source* directory.
+
+To load a **DirSheet**:
+
+- provide VisiData a directory as input (e.g. [:code]vd sample_data/[/])
+- {help.commands.open_dir_current} (equivalent to [:code]vd .[/])
+
+Pass the flag [:code]-r[/] ([:code]--recursive[/]) to display all of the files in all subfolders.
+
+Basic usage of the **DirSheet** is to find and load files within VisiData:
+
+- {help.commands.open_row_file}
+- {help.commands.open_rows}
+- ([:longname]open-dir-parent[/]) to open parent directory
+- {help.commands.sysopen_row}
+
+Advanced usage of **DirSheet** is as a file systems manager.
+
+- {help.commands.copy_row}
+- {help.commands.copy_selected}
+- [:keys]d[/] ([:longname]delete-row[/]) to delete file from filesystem
+- [:keys]e[/] ([:longname]edit-cell[/]) to change file metadata (all of the file metadata, except for *filetype*, is modifiable)
+
+Modifications on the **DirSheet** are deferred - they do not take effect on the filesystem itself until they are confirmed.
+- [:keys]z Ctrl+S[/] ([:longname]commit-sheet[/]) to commit all changes to the filesystem (not undoable!)
```

Comment:
why not "{help.commands.commit_sheet}"?

Reply:
ah I think maybe we should experiment with making commit-sheet an identical command but with a different helpstr on this sheet